README.md

```diff
@@ -40,6 +40,37 @@ downloader = HTTPDatasetDownloader()
 downloader.download(source_uri=source_uri, output=data_root)
 
 ```
+### Dataset Explore
+You can explore the downloaded dataset [schema](https://datasetinsights.readthedocs.io/en/latest/Synthetic_Dataset_Schema.html#synthetic-dataset-schema) by using following API:
```

Comment:
Users don't need to download datasets in order to use this package. This package should also work for development use cases when they generated the dataset on their workstation.